.github/workflows/doc.yml

```diff
@@ -48,7 +48,7 @@ jobs:
     - name: Build docs
       run: |
         cd docs
-        make -C . html-no-examples
+        make -C . html
```

Comment:
Change the name of the workflow to Doc only

Reply:
Good catch, thanks!